stubs/controller.policy-with-model.stub

```diff
@@ -0,0 +1 @@
+        $this->authorize('{{ method }}', {{ modelVariable }});
```

Comment:
Stubs aren't typically single line statements. I think this might be better served within a `PolicyStatement` that with a standard method to output the appropriate `authorize` call.